### ISSUE TITLE
any simple way to change CDN server?

### ISSUE THREAD

**Issue by peyoot** [twostairs/paperwork#738]
ajax.googleapis.com and fonts.googleapis.com and some other google server are blocked by some region.
See: http://stackoverflow.com/questions/2607425/is-google-s-cdn-for-jquery-available-in-china

There's some CDN mirror, So I want to manually change google CDN to other CDN server. Is there some config file that I can simple change a file rather than dig out all codes in different path? This could be a feature request for user to manually set up backup CDN server.

**Comment by Liongold** [comment]
Can you indicate the URL where you get the calls for ajax.googleapis.com and fonts.googleapis.com, please? I found a reference for ajax.googleapis.com in the Setup Wizard and will be replacing it. However, I did not find any other links? Once Paperwork is installed, it should not require any external dependency. If it still is, then there's a bug.

**Comment by peyoot** [comment]
yes, it happens in setup wizard phase.
after installation and I open home page with Firefox it will goes to setup.php. 
But it seems in Firefox it doesn't work, maybe because of these jquery scripts fails. 
Here I open it in Chrome and grab these links as bellow.

![setupphase1](https://cloud.githubusercontent.com/assets/2165697/21600674/29ed1fa4-d1bb-11e6-9b8f-6a6fc80b7bd7.PNG)
![setupphase](https://cloud.githubusercontent.com/assets/2165697/21600675/2e7983e6-d1bb-11e6-9b2d-9cd615fa7f68.PNG)

**Comment by Liongold** [comment]
Thanks for the clarification. Pull request #739 should fix this problem. If you can't wait, you can apply that patch to your install.

**Comment by Liongold** [comment]
The pull request has now been merged. Can you update your install and re-check to see if the bug has been fixed? Thanks for your help.

**Comment by Liongold** [comment]
Since no has reconfirmed this issue since a pull request fixing this issue has been merged, this issue will be closed. Thanks for your feedback. If you encounter any new issue with Paperwork, please open a new issue.
